libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -102,6 +108,16 @@ class CppInterface : public testing::Test {
     return module.GetErrorMessage();
   }
 
+  // Compiles a shader, expects compilation success, and returns the output
+  // bytes.
+  std::string CompilationOutput(const std::string& shader,
```

Comment:
In SPIR-V Tools tests we've found it much more useful to return this as a vector of uint32_t.
We had a bunch of utilities in the test code for manually creating vectors that matched SPIR-V instructions, then we could check element subsequence equality.
Please change this interface to return a vector of uint32_t.